### ISSUE TITLE
Recette from marmiton.org not fully scrapped

### ISSUE THREAD

**Issue by FrouxBY** [hay-kot/mealie#196]
Hi, 

Mealie is looking awesome, and I'm sure it will be soon a great tool for a lot of people, thanks for your work.

I try adding a recipes from marmiton.org: [this one for example](https://www.marmiton.org/recettes/recette_bechamel_23880.aspx) and it dit not get all the available information. 
Especially prepTime, cookTime and totalTime where missing.
I have tested using directly  the scrape_schema_recipe library, and it get those information, so it might be an issue when loading in the recipe model on your side

In addition (but less an issue, more feature request), the schema.org/recipe provide recipeCategory, keywords, aggregateRating that could directly populate respectivaly categories, keywords and rating as default when importing the recipe
Finally I will add 2 more features suggestions, fell free to open dedicated issue if you prefer: 
- add a list of tools required to perform the recipe in additon to ingredients
-   add a waiting time, this could help for recipe where some step are just waiting and not preparing or cooking.

Thanks again

**Comment by hay-kot** [comment]
> I try adding a recipes from marmiton.org: this one for example and it dit not get all the available information.
Especially prepTime, cookTime and totalTime where missing.

I was able to get this working. There was a bug in the cleaning process of the strings, but I'm able import this recipe and get the proper times. 

> In addition (but less an issue, more feature request), the schema.org/recipe provide recipeCategory, keywords, aggregateRating that could directly populate respectivaly categories, keywords and rating as default when importing the recipe

I have thought about this. My ultimate decision was that when creating my own collection of recipes I didn't really wan't the websites data, especially where some have like 50 categories and a rating I may not agree with. This may come in the future as a checkbox to check while importing, I'm always hesitant to add too many options and then maintaining becomes a mess. 

> add a list of tools required to perform the recipe in additon to ingredients
add a waiting time, this could help for recipe where some step are just waiting and not preparing or cooking.

Good Ideas, I've added them to the megathread, Thanks!
